<ui version="4.0" >
 <class>wndRun</class>
 <widget class="QMainWindow" name="wndRun" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>eudaq Run Control</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../images/Icon_euRun.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>4</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="grpControl" >
      <property name="title" >
       <string>Control</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="lblConfig" >
         <property name="text" >
          <string>Config: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lblRunmsg" >
         <property name="text" >
          <string>Run: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QPushButton" name="btnLog" >
         <property name="text" >
          <string>Log</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="txtLogmsg" />
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="txtRunmsg" />
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="cmbConfig" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable" >
          <bool>true</bool>
         </property>
         <property name="maxVisibleItems" >
          <number>16</number>
         </property>
         <property name="insertPolicy" >
          <enum>QComboBox::InsertAlphabetically</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QPushButton" name="btnStart" >
         <property name="text" >
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lblLogmsg" >
         <property name="text" >
          <string>Log: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="btnConfig" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Config</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QPushButton" name="btnStop" >
         <property name="text" >
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QPushButton" name="btnReset" >
         <property name="text" >
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpStatus" >
      <property name="title" >
       <string>Status</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpConnections" >
      <property name="title" >
       <string>Connections</string>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="viewConn" >
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>349</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnConfig</tabstop>
  <tabstop>txtRunmsg</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>txtLogmsg</tabstop>
  <tabstop>btnLog</tabstop>
  <tabstop>viewConn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>txtRunmsg</sender>
   <signal>returnPressed()</signal>
   <receiver>btnStart</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>247</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel" >
     <x>299</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtLogmsg</sender>
   <signal>returnPressed()</signal>
   <receiver>btnLog</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>238</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
